x-pack/plugins/fleet/public/applications/integrations/hooks/use_local_search.tsx

```diff
@@ -13,8 +13,11 @@ import type { IntegrationCardItem } from '../sections/epm/screens/home';
 export const searchIdField = 'id';
 export const fieldsToSearch = ['name', 'title', 'description'];
 
-export function useLocalSearch(packageList: IntegrationCardItem[]) {
+export function useLocalSearch(packageList: IntegrationCardItem[], isLoading: boolean) {
```

Comment:
I made some change to not index things to search while we are loading package as it seems to have some performance impact